vnext/JSI/Universal/ChakraRtRuntime.h

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#ifndef CHAKRACORE
+
+#include "ChakraRuntime.h"
+#include "ChakraRuntimeArgs.h"
+
+#include <memory>
+#include <string>
+
+namespace Microsoft::JSI {
+
+class ChakraRtRuntime : public ChakraRuntime {
+ public:
+  using ChakraRuntime::ChakraRuntime;
+  ~ChakraRtRuntime() noexcept = default;
+
+#pragma region Functions_inherited_from_Runtime
+
+  facebook::jsi::Value evaluatePreparedJavaScript(
+      const std::shared_ptr<const facebook::jsi::PreparedJavaScript> &preparedJs) override;
+
+  std::string description() override;
+
+ private:
+  // Only ChakraCore supports weak reference semantics, so ChakraRtRuntime
+  // WeakObjects are in fact strong references.
+  facebook::jsi::WeakObject createWeakObject(const facebook::jsi::Object &obj) override;
```

Comment:
>override [](start = 79, length = 8)

protected?

Reply:
See other comment.

---
In reply to: [345444502](https://github.com/microsoft/react-native-windows/pull/3650#discussion_r345444502) [](ancestors = 345444502)